### DISCUSSION TITLE
Deleting Volumes?

### DISCUSSION THREAD

**Discussion by dragon2611** [seaweedfs/seaweedfs#1850] [Q&A]
Probably A stupid question, But if I delete a volume does it delete the files contained within or do they get re-balanced onto another volume?.

**Comment by chrislusf** [comment]
Assuming you are talking about master+volume servers:

- if the volume data is not replicated, the files will be gone.
- if the volume data is replicated, you can run `volume.fix.replication` in `weed shell` to re-create the missing copy.
